scripts/calculateKpi/protocols/tetherV0/index.ts

```diff
@@ -0,0 +1,149 @@
+)
+
+// Token addresses from https://www.coingecko.com/en/coins/tether, https://www.coingecko.com/en/coins/usdt0, https://docs.inkonchain.com/useful-information/ink-contracts
+const networkToTokenAddress: Partial<Record<NetworkId, Address>> = {
+  [NetworkId['ethereum-mainnet']]: '0xdac17f958d2ee523a2206206994597c13d831ec7',
+  [NetworkId['avalanche-mainnet']]:
+    '0x9702230a8ea53601f5cd2dc00fdbc13d4df4a8c7',
+  [NetworkId['celo-mainnet']]: '0x48065fbBE25f71C9282ddf5e1cD6D6A887483D5e',
+  [NetworkId['unichain-mainnet']]: '0x9151434b16b9763660705744891fa906f660ecc5',
+  // [NetworkId['sei-mainnet']]: '0x9151434b16b9763660705744891fa906f660ecc5',
+  [NetworkId['ink-mainnet']]: '0x0200C29006150606B650577BBE7B6248F58470c1',
+  [NetworkId['op-mainnet']]: '0x01bff41798a0bcf287b996046ca68b395dbc1071',
+  [NetworkId['arbitrum-one']]: '0xfd086bc7cd5c481dcc9c85ebe478a1c0b69fcbb9',
+  [NetworkId['berachain-mainnet']]:
+    '0x779ded0c9e1022225f8e0630b35a9b54be713736',
+}
+
+async function getEligibleTxCount({
+  networkId,
+  user,
+  startBlock,
+  endBlockExclusive,
+  tokenAddress,
+}: {
+  networkId: NetworkId
+  user: Address
+  startBlock?: number
+  endBlockExclusive?: number
+  tokenAddress: Address
+}): Promise<number> {
+  const client = getHyperSyncClient(networkId)
+
+  // Since we are using events to count transfers, we need to ensure that transactions that emit multiple events are only counted once
+  const transactionHashesCounted = new Set<string>()
+  let totalTransactions = 0
+
+  const query = {
+    transactions: [{ from: [user] }],
+    logs: [
+      {
+        contractAddress: tokenAddress,
+        topics: [[transferEventSigHash], [pad(user, { size: 32 })], [], []],
+      },
+      {
+        contractAddress: tokenAddress,
+        topics: [[transferEventSigHash], [], [pad(user, { size: 32 })], []],
+      },
+    ],
+    fieldSelection: {
+      log: [
+        LogField.Data,
+        LogField.Address,
+        LogField.Topic0,
+        LogField.Topic1,
+        LogField.Topic2,
+        LogField.Topic3,
+        LogField.TransactionHash,
+      ],
+      transaction: [TransactionField.Hash],
+    },
+    fromBlock: startBlock ?? 0,
+    ...(endBlockExclusive && { toBlock: endBlockExclusive }),
+  }
+
+  await paginateQuery(client, query, async (response) => {
+    for (const { data, topics, transactionHash } of response.data.logs) {
+      if (data && transactionHash) {
+        const decodedLog = decodeEventLog({
+          abi: erc20Abi,
+          data: data as Hex,
+          topics: topics as [],
+        })
+        const transferValue = decodedLog.args.value
+
+        if (
+          BigNumber(transferValue).gt(MIN_ELIGIBLE_VALUE_IN_SMALLEST_UNIT) &&
+          !transactionHashesCounted.has(transactionHash)
+        ) {
+          totalTransactions += 1
+          transactionHashesCounted.add(transactionHash)
+        }
+      }
+    }
+  })
+
+  return totalTransactions
+}
+
+export async function calculateKpi({
```

Comment:
nit: we should document this function like we did for the other campaigns (KPI unit, business purpose, calculation method, etc).